ppr-ui/src/components/search/SearchBar.vue

```diff
@@ -87,7 +87,7 @@
                   filled
                   :hint="searchHintFirst"
                   persistent-hint
-                  :label="optionFirst"
+                  label="First Name"
```

Comment:
Just to confirm, the _Optional_ is removed from all roles? I remember it was added recently, and now removed?

Reply:
Ticket called for it to be removed from staff accounts, it was already removed from the other roles

Reply:
Will double check though, does seem a bit odd

Reply:
Great, thank you!